tests/testthat/test-build-home-index.R

```diff
@@ -25,3 +25,12 @@ test_that("package repo verification", {
   )
 })
 
+test_that("page title",{
+  pkg <- test_path("assets/pagetitle")
+  expect_equal(as.character(data_home(pkg))[[1]],
```

Comment:
Can you please use tidyverse style for long lines here? http://style.tidyverse.org/syntax.html#long-lines